Makefile

```
@@ -30,6 +30,7 @@ include/sourmash.h: src/core/src/lib.rs \
                     src/core/src/ffi/minhash.rs \
                     src/core/src/ffi/signature.rs \
                     src/core/src/ffi/nodegraph.rs \
+					src/core/src/ffi/lca_db.rs \
```

Nitpick: replace tabs with spaces here =]
```suggestion
                    src/core/src/ffi/lca_db.rs \
```